po/zh_CN.po

```diff
@@ -495,228 +495,233 @@ msgstr ""
 #: assign.c:1074
 #, c-format
 msgid "Unsupported column type in assign.c:memrecycle '%s'"
-msgstr ""
+msgstr "assign.c:memrecycle '%s' 里有不支持的列的类型"
 
 #: assign.c:1121
 #, c-format
 msgid "Internal error: writeNA passed a vector of type '%s'"
-msgstr ""
+msgstr "内部错误：writeNA 函数读取到了一个类型是'%s'的向量"
 
 #: assign.c:1152
 #, c-format
 msgid ""
 "Internal error: savetl_init checks failed (%d %d %p %p). please report to "
 "data.table issue tracker."
-msgstr ""
+msgstr "内部错误：savetl_init的校验失败 (%d %d %p %p)，请将此问题汇报给"
+"data.table 问题追踪器。"
 
 #: assign.c:1160
 #, c-format
 msgid "Failed to allocate initial %d items in savetl_init"
-msgstr ""
+msgstr "不能给 savetl_init 最开始的 %d 个项进行分配 "
 
 #: assign.c:1169
 #, c-format
 msgid ""
 "Internal error: reached maximum %d items for savetl. Please report to data."
 "table issue tracker."
-msgstr ""
+msgstr "内部错误：已经达到了savetl能处理的最多 %d 数量的子项。请将此问题汇报给"
+"data.table 问题追踪器。"
 
 #: assign.c:1176
 #, c-format
 msgid "Failed to realloc saveds to %d items in savetl"
-msgstr ""
+msgstr "不能给savetl里的 %d 个项提供 saveds"
 
 #: assign.c:1182
 #, c-format
 msgid "Failed to realloc savedtl to %d items in savetl"
-msgstr ""
+msgstr "不能给savetl里的 %d 个项提供 savetl"
 
 #: assign.c:1205
 msgid "x must be a character vector"
-msgstr ""
+msgstr "x 必须是一个字符向量"
 
 #: assign.c:1206
 msgid "'which' must be an integer vector"
-msgstr ""
+msgstr "'which' 必须是一个整数向量"
 
 #: assign.c:1207
 msgid "'new' must be a character vector"
-msgstr ""
+msgstr "'new' 必须是一个字符向量"
 
 #: assign.c:1208
 #, c-format
 msgid "'new' is length %d. Should be the same as length of 'which' (%d)"
-msgstr ""
+msgstr "'new' 的长度是 %d。 它的长度必须和'which' (%d)的长度一致。"
 
 #: assign.c:1211
 #, c-format
 msgid ""
 "Item %d of 'which' is %d which is outside range of the length %d character "
 "vector"
 msgstr ""
+"'which' 的 %1%d 项是 %2%d 且超出了长度为%3%d字符的范围"
```

Comment:
A bit awkward I know, but the correct way is `%1$d`, etc, I think that applies to a few of the others as well

Reply:
I've just changed them all. Pushing them now. :)